presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/StaticMetastoreLocator.java

```diff
@@ -63,14 +70,19 @@ public StaticMetastoreLocator(StaticMetastoreConfig config, ThriftMetastoreAuthe
     }
 
     public StaticMetastoreLocator(List<URI> metastoreUris, @Nullable String metastoreUsername, ThriftMetastoreClientFactory clientFactory)
+    {
```

Comment:
the `public StaticMetastoreLocator(List<URI> metastoreUris, @Nullable String metastoreUsername, ThriftMetastoreClientFactory clientFactory)` seems unused now